A few minor Debian customizations of the HTML PETSc documentation.
Remove links to non-free PDF files.

Index: petsc-3.4.2/docs/bugreporting.html
===================================================================
--- petsc-3.4.2.orig/docs/bugreporting.html	2013-08-20 15:24:54.000000000 +0200
+++ petsc-3.4.2/docs/bugreporting.html	2013-08-20 15:24:54.000000000 +0200
@@ -59,7 +59,7 @@
 
         <li>
           PETSc Version: (run PETSc program with -v, or look in
-          <a href="petscversion.h">include/petscversion.h</a>)
+          <a href="/usr/lib/petscdir/3.3/include/petscversion.h">include/petscversion.h</a>)
         </li>
 
         <li>MPI implementation: (e.g., MPICH, LAM, IBM, SGI)</li>
Index: petsc-3.4.2/docs/index.html
===================================================================
--- petsc-3.4.2.orig/docs/index.html	2013-08-20 15:24:54.000000000 +0200
+++ petsc-3.4.2/docs/index.html	2013-08-20 15:24:54.000000000 +0200
@@ -23,7 +23,7 @@
       </p>
 
       <ul>
-        <li><b>PETSc users manual</b> - <a href="manual.pdf">pdf (fully searchable with hyperlinks)</a></li>
+        <li><b>PETSc users manual</b> - <a href="http://www.mcs.anl.gov/petsc/petsc-current/docs/manual.pdf">pdf (fully searchable with hyperlinks)</a></li>
         <li><a href="manualpages/help.html">Index</a> of all help topics</li>
         <li><a href="manualpages/singleindex.html">Index</a> of all manual pages</li>
         <li><a href="manualpages/Sys/UsingFortran.html">Strategies</a> for using PETSc with Fortran</li>
@@ -119,6 +119,7 @@
         <li><a href="codemanagement.html">Code Management</a> - hints for efficient management of large-scale application codes</li>
         <li><a href="faq.html">FAQ</a> - frequently asked questions about PETSc</li>
         <li><a href="copyright.html">License</a> - statement of the PETSc copyright</li>
+        <li><a href="../README.Debian.gz">README.Debian</a> - a description of this Debian package, see also the<a href="http://lyre.mit.edu/~powell/petsc.html">package homepage</a></li>
       </ul>
 
     </div>
Index: petsc-3.4.2/docs/faq.html
===================================================================
--- petsc-3.4.2.orig/docs/faq.html	2013-08-20 15:24:54.000000000 +0200
+++ petsc-3.4.2/docs/faq.html	2013-08-20 15:24:54.000000000 +0200
@@ -439,23 +439,22 @@
 
       <ol>
         <li>
-          The directory ${PETSC_DIR}/docs contains a set of HTML manual pages in
-          for use with a browser. You can delete these pages to save about .8
+          The directory /usr/share/doc/petsc3.3-doc contains a set of HTML manual pages in
+          for use with a browser. Don't install the -doc package to save about .8
           Mbyte of space.
         </li>
 
         <li>
           The PETSc users manual is provided in PDF in
-          ${PETSC_DIR}/docs/manual.pdf. You can delete this.
+          <a href="http://www.mcs.anl.gov/petsc/petsc-current/docs/manual.pdf">manual.pdf</a>.
         </li>
 
         <li>
           The PETSc test suite contains sample output for many of the examples.
           These are contained in the PETSc directories
-          ${PETSC_DIR}/src/*/examples/tutorials/output and
-          ${PETSC_DIR}/src/*/examples/tests/output. Once you have run the test
-          examples, you may remove all of these directories to save about 300
-          Kbytes of disk space.
+          /usr/share/doc/petsc3.3-doc/src/*/examples/tutorials/output and
+          /usr/share/doc/petsc3.3-doc/src/*/examples/tests/output. Don't install
+	  the -doc package to save disk space.
         </li>
 
         <li>
